packages/quantic/force-app/examples/main/lwc/exampleQuanticGeneratedAnswer/exampleQuanticGeneratedAnswer.js

```diff
@@ -29,6 +29,13 @@ export default class ExampleQuanticGeneratedAnswer extends LightningElement {
         'Indicates whether footer sections should be displayed on multiple lines.',
       defaultValue: false,
     },
+    {
+      attribute: 'withToggle',
+      label: 'With Toggle',
+      description:
+        'Indicates whether the component should render the toggle button.',
```

Comment:
very minor, but for me this gives more precision.
```suggestion
        'Indicates whether the generated answer can be toggled on or off,
```